lambda/getProductsList/index.ts

```diff
@@ -0,0 +1,32 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+
+export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
+  try {
+    // Mock data for the products array
+    const products = [
+      { id: '1', title: 'Milk', description: 'Some text', price: 3, count: 5 },
+      { id: '2', title: 'Bread', description: 'Some text', price: 6, count: 4 },
+      { id: '3', title: 'Eggs', description: 'Some text', price: 7, count: 3 },
+    ];
+
+    return {
+      statusCode: 200,
```

Comment:
It would be better to create separate utils for the same logic. For example, a function that creates an object with headers and a status code.